compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -1524,13 +1524,21 @@ extern "C" void LLVMRustFreeOperandBundleDef(OperandBundleDef *Bundle) {
 
 extern "C" LLVMValueRef LLVMRustBuildCall(LLVMBuilderRef B, LLVMTypeRef Ty, LLVMValueRef Fn,
                                           LLVMValueRef *Args, unsigned NumArgs,
-                                          OperandBundleDef **OpBundles,
+                                          OperandBundleDef **OpBundlesIndirect,
                                           unsigned NumOpBundles) {
   Value *Callee = unwrap(Fn);
   FunctionType *FTy = unwrap<FunctionType>(Ty);
+
+  // FIXME: Is there a way around this?
+  std::vector<OperandBundleDef> OpBundles;
```

Comment:
it's not a problem that the vector is deallocated when returning?

Reply:
Shouldn't be, as far as I can tell, the ArrayRef types makes no promises of the memory region existing past the duration of the call.

Reply:
of course... `CreateCall` should be aware that it must copy the data of the `ArrayRef` if it needs to store it.